lib/omnibus/cli.rb

```diff
@@ -70,6 +70,10 @@ def execute!
       desc: "Use the given manifest when downloading software sources.",
       type: :string,
       default: nil
+    method_option :populate_omnibus_cache,
```

Comment:
Can we call this `populate_s3_cache`. The Omnibus bit is obvious as we are using the `omnibus` binary!